content/en/quality_gates/search/_index.md

```diff
@@ -15,7 +15,7 @@ further_reading:
 
 The [**Quality Gate Rules** page][1] is useful for developers who want to keep an eye on the quality gates for their build pipelines. You can see all of the rules defined by the organization.
 
-{{< img src="quality_gates/rules_list_1.png" alt="Quality Gate rules in Datadog displaying the rule name, evaluation, scope, blocking status, last modified date, and the creator avatar" style="width:100%" >}}
+{{< img src="quality_gates/rules_list_2.png" alt="Quality Gate rules in Datadog displaying the rule name, evaluation, scope, blocking status, last modified date, and the creator avatar" style="width:100%" >}}
```

Comment:
```suggestion
{{< img src="quality_gates/setup/rules_list_2.png" alt="Quality Gate rules in Datadog displaying the rule name, evaluation, scope, blocking status, last modified date, and the creator avatar" style="width:100%" >}}
```

Reply:
The image link is broken. I believe this is the correct path to the image.

Reply:
Same comment here, I'll move the file to the correct location